From f44580a423099c046df5a05107b6c5b21368bb9e Mon Sep 17 00:00:00 2001
From: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Date: Wed, 7 Apr 2021 14:05:50 +0300
Subject: [PATCH 28/42] s32-gen1: flashmap: Update Linux dtb + rootfs offsets

This update is needed because on Linux Kernel version 5.10,
the size of the Kernel image has increased from ~10MB to ~13MB.

Upstream-Status: Pending

Issue: ALB-6731
Signed-off-by: Ciprian Marian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/configs/s32.h b/include/configs/s32.h
index 6c3baca418..7e42eae2c0 100644
--- a/include/configs/s32.h
+++ b/include/configs/s32.h
@@ -143,15 +143,15 @@
  * 'conf/machine/include/s32*flashmap.inc'.
  */
 #define CONFIG_SYS_FLASH_BASE		CONFIG_SYS_FSL_FLASH0_BASE
-#define KERNEL_FLASH_MAXSIZE		0xa00000
+#define KERNEL_FLASH_MAXSIZE		0xd00000
 #define FDT_FLASH_MAXSIZE		0x100000
 #define RAMDISK_FLASH_MAXSIZE		0x2000000
 #define UBOOT_FLASH_ADDR		(CONFIG_SYS_FSL_FLASH0_BASE + 0x0)
 
 #ifdef CONFIG_S32_GEN1
 #  define KERNEL_FLASH_ADDR	(CONFIG_SYS_FSL_FLASH0_BASE + 0x1f0000)
-#  define FDT_FLASH_ADDR	(CONFIG_SYS_FSL_FLASH0_BASE + 0xbf0000)
-#  define RAMDISK_FLASH_ADDR	(CONFIG_SYS_FSL_FLASH0_BASE + 0xcf0000)
+#  define FDT_FLASH_ADDR	(CONFIG_SYS_FSL_FLASH0_BASE + 0xef0000)
+#  define RAMDISK_FLASH_ADDR	(CONFIG_SYS_FSL_FLASH0_BASE + 0xff0000)
 #else
 #  define KERNEL_FLASH_ADDR	(CONFIG_SYS_FSL_FLASH0_BASE + 0x100000)
 #  define FDT_FLASH_ADDR	(CONFIG_SYS_FSL_FLASH0_BASE + 0xb00000)
-- 
2.25.1

